### PR TITLE
Added declaration file package

### PR DESCRIPTION
**Pull request by Liron-Toledo**
Added declaration file package to repo AND added an import to the package in the CLI plugin template

### CODE REVIEW

File: vatomcli-plugin-template/src/index.js [hunk 1]
```diff
@@ -1,3 +1,8 @@
+import { BasePlugin } from 'vatom-spaces-plugins'
+import { BaseComponent } from 'vatom-spaces-plugins'
```
Comment (chapmankyle) [inline vatomcli-plugin-template/src/index.js]: You could reduce this to
```suggestion
import { BasePlugin, BaseComponent } from 'vatom-spaces-plugins'
```

File: vatomcli-plugin-template/src/index.js [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+
+BasePlugin
```
Comment (chapmankyle) [inline vatomcli-plugin-template/src/index.js]: Is this supposed to be here?

Comment (Liron-Toledo) [reply vatomcli-plugin-template/src/index.js]: OMG! that is so stupid of me. No that shouldn't be there. Removed in last push. Sorry.

File: vatomcli-plugin-template/src/index.js (hunk 1 above)
Comment (Liron-Toledo) [reply vatomcli-plugin-template/src/index.js]: Ya you're right. Shortened to one line in last push

**Review by chapmankyle — APPROVED** [review]

**Pull request merged**